Documentation/Updating.md

```diff
@@ -1,9 +1,14 @@
 # Updating the Microsoft Mixed Reality Toolkit
 
+- [How to update](#How-to-update-your-project-to-use-a-new-version-of-MRTK)
```

Comment:
It looks like this link is broken which caused the docs warning.

Reply:
The link works when editing in VSCode - my only guess is that it's too long.  I'll try shortening.